js/components/scrubberBar.js

```diff
@@ -10,32 +10,13 @@ var React = require('react'),
 var ScrubberBar = React.createClass({
   mixins: [ResizeMixin],
 
-  getDefaultProps: function () {
-    return {
-      skinConfig: {
-        controlBar: {
-          scrubberBar: {
-            backgroundColor: 'rgba(5,175,175,1)',
-            bufferedColor: 'rgba(127,5,127,1)',
-            playedColor: 'rgba(67,137,5,1)'
-          },
-          adScrubberBar: {
-            backgroundColor: 'rgba(175,175,5,1)',
-            bufferedColor: 'rgba(127,5,127,1)',
-            playedColor: 'rgba(5,63,128,1)'
-          }
-        }
-      }
-    };
-  },
-
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
-    this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
-    this.lastScrubX = null;
```

Comment:
lastScrubX is just bookkeeping, does it need to be in the React state? I don't think there's a reason to trigger a redraw when it changes.